packages/wrangler/src/deployment-bundle/node-compat.ts

```diff
@@ -0,0 +1,103 @@
+import { UserError } from "../errors";
+import { logger } from "../logger";
+
+export type NodeJSCompatMode = "legacy" | "v1" | "v2" | null;
```

Comment:
Feel free to ignore this, but what about naming this with a bit more verbosity to describe what they do? Something like `legacy-build-time`/`runtime-only`/`hybrid-build-and-runtime`

Reply:
I see where you are coming from but I think the more verbose strings will actually make the code less readable.
I'll add a JSDOC comment to explain the values here.

Reply:
https://github.com/cloudflare/workers-sdk/pull/5991/commits/852db02ad3c8ff499c5910180c037b5a137f2f1d